packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/LeftToolbar.module.less

```diff
@@ -20,21 +20,19 @@
   .toolbar();
   .vertical-toolbar();
 
+  overflow: visible;
   width: 32px;
-  margin-top: 8px;
-  margin-left: 4px;
+  margin-top: 12px;
+  margin-left: 14px;
 
   // TODO: change group style, if will decide to get rid of delimiter everywhere
   .group:not(:last-of-type) {
-    margin-bottom: 0;
     border-bottom: none;
-    padding-bottom: 0;
   }
-
 }
 
 .buttons {
-  overflow: hidden;
+  overflow: visible;
```

Comment:
Could you clarify, why `overflow` is required?

Reply:
toolbars shadow clipped if overflow: hidden